architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,540 @@
+    clientConfig: <4>
+      service:
+        namespace: default  <5>
+        name: kubernetes <6>
+       path: <webhook_url> <7>
+      caBundle: <cert> <8>
+    rules: <9>
+    - operations: <10>
+      - <operation>
+      apiGroups:
+      - ""
+      apiVersions:
+      - "*"
+      resources:
+      - <resource>
+    failurePolicy: <policy> <11>
+----
+
+<1> The name for the admission webhook controller.
+<2> The name for the webhook.
+<3> Specify information about how to connect to, trust, and send data to the webhook admission server.
+<4> The project where the front-end service is created.
+<5> The name of the front-end service.
+<6> The webhook URL used for admission requests.
+<7> A PEM-encoded CA certificate that signs the server certificate used by the webhook.
+<8> Rules that define when the API server should use this plug-in.
+<9> The operation that triggers the API server to call this plug-in. In the example the API server will call 
+this plug-in upon creation of pods.
+* create
+* update
+* delete
+* connect
+<10> Specify how the policy should proceed if the webhook admission server is unavailable. 
+Either `Ignore` (allow/fail open) or `Fail` (block/fail closed). 
+
+[[architecture-additional-concepts-dynamic-admission-webhooks-ex-v]]
+//http://blog.kubernetes.io/2018/01/extensible-admission-is-beta.html
+.Sample validating admission webhook controller configuration:
+[source-yaml]
+----
+apiVersion: admissionregistration.k8s.io/v1beta1
+  kind: ValidatingWebhookConfiguration <1>
+  metadata:
+    name: <controller_name> <2>
+  webhooks:
+  - name: <webhook_name> <3>
+    clientConfig: <4>
+      service:
+        namespace: default  <5>
+        name: kubernetes <6>
+       path: <webhook_url> <7>
+      caBundle: <cert> <8>
+    rules: <9>
+    - operations: <10>
+      - <operation>
+      apiGroups:
+      - ""
+      apiVersions:
+      - "*"
+      resources:
+      - <resource>
+    failurePolicy: <policy> <11>
+----
+
+<1> The name for the admission webhook controller.
+<2> The name for the webhook.
+<3> Specify information about how to connect to, trust, and send data to the webhook admission server.
+<4> The project where the front-end service is created.
+<5> The name of the front-end service.
+<6> The webhook URL used for admission requests.
+<7> A PEM-encoded CA certificate that signs the server certificate used by the webhook.
+<8> Rules that define when the API server should use this plug-in.
+<9> The operation that triggers the API server to call this plug-in. In the example the API server will call 
+this plug-in upon creation of pods.
+* create
+* update
+* delete
+* connect
+<10> Specify how the policy should proceed if the webhook admission server is unavailable. 
+Either `Ignore` (allow/fail open) or `Fail` (block/fail closed). 
+
+[NOTE]
+====
+Fail open can result in unpredictable behavior for all clients.
+====
+
+Whenever an API request comes in, the admission controller 
```

Comment:
what is "the admission controller"? I guess you mean the Mutating/ValidatingAdmissionWebhook.